build-vm-zvm

```diff
@@ -21,17 +21,18 @@
 #
 ################################################################
 
+ZVM_CLEANUP=
+ZVM_VOLUME_ROOT=${ZVM_VOLUME_ROOT:-0100}
+ZVM_VOLUME_SWAP=${ZVM_VOLUME_SWAP:-0200}
+
 # z/VM: use default kernel image from local machine
```

Comment:
actually why 0100 and not 0150, wasn't 0150 the default root dev ?

Reply:
Good question. we cn change it back, I just found that most people do not use 150 but 100 as base address